extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-validation.service.ts

```diff
@@ -127,7 +127,7 @@ export class DragAndDropSortInputValidationService {
         var rule = rules[j];
         if (!customizationArgs.allowMultipleItemsInSamePosition.value) {
           var xInputs = inputs.x as string[][];
-          for (var k = 0; k < xInputs.length; k++) {
+          for (var k = 0; xInputs && k < xInputs.length; k++) {
```

Comment:
Hi @Shivkant-Chauhan, I'm a bit confused as to how this fixes the errors..

Were you able to reproduce the original issue on your local machine? Could you provide the exact steps you followed to test the fix here? Thanks!

Reply:
yes, the issue was reproducible on my local server. there were 2 errors, one was some nonStrictValidationFaliure and I still can't figure out how to resolve that. and the other one was that somewhere we were accessing the length property of a null element. from the error logs I tried finding this element in the file and in any case, the xInputs was NULL in this for loop, so I put a check that xInputs must not be NULL before accessing its length property. this fixes the `TypeError: Cannot read properties of NULL (reading 'length')`

Reply:
Could you please provide the steps you used to reproduce the error?

Reply:
- create a new exploration and add a drag and drop interaction (with at least 2 items!) and save the interaction.
- on the exploration editor page, try adding an extra rule for the interaction by clicking the '+ Add Another Possible Answer' 
![image](https://user-images.githubusercontent.com/91013793/206545855-3cbfe598-ded2-42ff-91f0-28e291e2c46a.png)
- on saving the new rule spec, you will get all the errors in the console.. although all of them got fixed by this fixing the length error changes in this PR.. but please help me in fixing that one validation error..

Reply:
@kevintab95 I just got to know that the issue is reproducible when the first rule spec is: 'has ordering equal to' in the drag and drop interaction